docs/list_of_tutorials.jl

```diff
@@ -65,6 +67,7 @@ if generate_tutorials
         "Atmos/dry_rayleigh_benard.jl",               # takes too long
         "Atmos/heldsuarez.jl",                        # broken
         "Atmos/risingbubble.jl",                      # broken
+        "Atmos/burgers_single_stack.jl",              # too long
```

Comment:
@jakebolewski 😢

Reply:
> How long does it take?
🤷‍♂️ @ilopezgp ? But there are several more that will extend the overall doc build (#1219, #1226, and more from #1181).

> can you open an issue for this?

Opened #1233

Reply:
> How long does it take?

Now it takes 4 minutes running on CPU on my iMac (4.2 GHz Intel Core i7, 16GB RAM). I was not sure if that was too much time so I added the case to the skip list.

Reply:
Got it. I will remove it from the skip list.